app/utils/answersHelpers/ConsignmentAnswersHelper.scala

```diff
@@ -29,13 +30,60 @@ import viewModels.sections.Section.{AccordionSection, StaticSection}
 
 class ConsignmentAnswersHelper(userAnswers: UserAnswers)(implicit messages: Messages) extends AnswersHelper(userAnswers) {
 
+  def transitOperationSection: StaticSection = StaticSection(
+    rows = Seq(reducedDatasetIndicatorRow) ++ securityTypeRow ++ declarationTypeRow ++ declarationAcceptanceDateRow
+  )
+
   def headerSection: Section = StaticSection(
     rows = Seq(
       customsOfficeOfDestinationActual,
       traderAtDestinationRow
     )
   )
 
+  def declarationTypeRow: Option[SummaryListRow] = userAnswers
+    .get(DeclarationTypePage)
+    .map(_.code)
```

Comment:
Why do we look this up in reference data if we're only showing the code?

Reply:
I originally thought that the page was to display the description not the code, similar to the security type. Will make changes

Reply:
removed